### PR TITLE
Issue 2396 Make Mary Era the default

### PR DESCRIPTION
**Pull request by newhoggy**
https://github.com/input-output-hk/cardano-node/issues/2396

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs [hunk 1]
```diff
@@ -1521,19 +1521,19 @@ pCardanoEra = asum
...
       <> Opt.help "Specify the Allegra era"
       )
   , Opt.flag' (AnyCardanoEra MaryEra)
       (  Opt.long "mary-era"
-      <> Opt.help "Specify the Mary era"
+      <> Opt.help "Specify the Mary era (default)"
       )
 
     -- Default for now:
-  , pure (AnyCardanoEra ShelleyEra)
```
Comment (Jimbo4350) [inline cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:1536]: :+1:

**Review by Jimbo4350 — APPROVED** [review]
LGTM! Just squash the commits before borsing/

**Review by krissemicolon — COMMENTED** [review]
LGTM 👍🏼

**Pull request merged**